test/e2e/case1_pod_handling_test.go

```diff
@@ -293,4 +295,66 @@ var _ = Describe("Test pod obj template handling", Ordered, func() {
 			utils.KubectlDelete("event", "--field-selector=involvedObject.name="+configPolicyName, "-n", "managed")
 		})
 	})
+
+	Describe("Test pod obj in "+ocmNs+" namespace", Ordered, func() {
+		const (
+			configPolicyNameInform  string = "policy-pod-create-inform"
+			configPolicyNameEnforce string = "policy-pod-create"
+			podName                 string = "nginx-pod-e2e"
+			policyYamlInform        string = "../resources/case1_pod_handling/case1_pod_create_inform.yaml"
+			policyYamlEnforce       string = "../resources/case1_pod_handling/case1_pod_create_enforce.yaml"
+		)
+
+		BeforeAll(func() {
+			utils.Kubectl("create", "ns", ocmNs)
+		})
+		It("should be created properly on the managed cluster", func() {
+			By("Creating " + policyYamlInform + " on managed")
+			utils.Kubectl("apply", "-f", policyYamlInform, "-n", ocmNs)
+			plc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigPolicy,
+				configPolicyNameInform, ocmNs, true, defaultTimeoutSeconds)
+			Expect(plc).NotTo(BeNil())
+			Eventually(func(g Gomega) {
+				managedPlc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigPolicy,
+					configPolicyNameInform, ocmNs, true, defaultTimeoutSeconds)
+
+				utils.CheckComplianceStatus(g, managedPlc, "NonCompliant")
+			}, defaultTimeoutSeconds, 1).Should(Succeed())
+		})
+		It("should create pod on managed cluster", func() {
+			By("creating " + policyYamlEnforce + " on hub with spec.remediationAction = enforce")
+			utils.Kubectl("apply", "-f", policyYamlEnforce, "-n", ocmNs)
+			plc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigPolicy,
+				configPolicyNameEnforce, ocmNs, true, defaultTimeoutSeconds)
+			Expect(plc).NotTo(BeNil())
+			Eventually(func(g Gomega) {
+				managedPlc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigPolicy,
+					configPolicyNameEnforce, ocmNs, true, defaultTimeoutSeconds)
+
+				utils.CheckComplianceStatus(g, managedPlc, "Compliant")
+			}, defaultTimeoutSeconds, 1).Should(Succeed())
+			Eventually(func(g Gomega) {
+				informPlc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigPolicy,
+					configPolicyNameInform, ocmNs, true, defaultTimeoutSeconds)
+
+				utils.CheckComplianceStatus(g, informPlc, "Compliant")
+			}, defaultTimeoutSeconds, 1).Should(Succeed())
+			pod := utils.GetWithTimeout(clientManagedDynamic, gvrPod,
+				podName, "default", true, defaultTimeoutSeconds)
+			Expect(pod).NotTo(BeNil())
+		})
+		AfterAll(func() {
+			utils.KubectlDelete("ns", ocmNs)
+			policies := []string{
+				configPolicyNameInform,
+				configPolicyNameEnforce,
+			}
+			deleteConfigPolicies(policies)
```

Comment:
Is this still needed since the namespace is deleted?

Reply:
Oh no